module-services/service-audio/ServiceAudio.cpp

```diff
@@ -492,7 +497,7 @@ std::unique_ptr<AudioResponseMessage> ServiceAudio::HandleSendEvent(std::shared_
         input.audio->SendEvent(evt);
     }
 
-    return std::make_unique<AudioEventResponse>(RetCode::Success);
+    return sys::msgHandled();
```

Comment:
doesn't it break anything?

Reply:
seems not. 
This message is now multicast, so there is no need to send a response to a sender. Also, I've checked and multicast messages are handled like this.